packages/data/src/plugins/actions.js

```diff
@@ -123,7 +124,7 @@ export function* activatePlugins( plugins ) {
 		} );
 
 		if ( results && results.status === 'success' ) {
-			yield updateActivePlugins( results.activatedPlugins );
+			yield updateActivePlugins( null, results.activatedPlugins );
```

Comment:
`updateActivePlugins` is also used in `with-plugins-hydration.js` and I wonder if active/added isn't clear. How about a `replace` flag that defaults to false? Something like that so the default is using `updateActivePlugins` to add items, but you can also replace it wholesale. Would that make more sense?

Reply:
I think you're right.  A `replace` flag makes it more clear.

Updated `updateActivePlugins` and `updateInstalledPlugins` to use this pattern.